src/components/icons/feather/User.tsx

```diff
@@ -1,33 +1,37 @@
 import React, { FC } from 'react';
-import { ViewStyle } from 'react-native';
+import { TouchableOpacity, ViewStyle } from 'react-native';
 import { Circle, Path, Svg } from 'react-native-svg';
 
 interface Props {
 	style?: ViewStyle;
 	size?: number;
 	color?: string;
+	onPress?: () => void;
 }
 
 export const UserIcon: FC<Props> = ({
 	style,
 	size = 24,
 	color = 'white',
+	onPress,
 }: Props) => {
 	return (
-		<Svg
-			style={style}
-			width={size}
-			height={size}
-			viewBox="0 0 24 24"
-			fill="none"
-			stroke={color}
-			strokeWidth="2"
-			strokeLinecap="round"
-			strokeLinejoin="round"
-		>
-			<Path d="M20 21v-2a4 4 0 0 0-4-4H8a4 4 0 0 0-4 4v2"></Path>
-			<Circle cx="12" cy="7" r="4"></Circle>
-		</Svg>
+		<TouchableOpacity onPress={onPress}>
```

Comment:
Should separate material components (i.e. images, svg icons, ...) from logic, handle function where using these material components.

Reply:
I've fixed, pls check again